src/devices/XRDevice.js

```diff
@@ -128,15 +135,17 @@ export default class XRDevice extends EventTarget {
    * @param {XREye} eye
    * @param {XRWebGLLayer} layer
    * @param {Object?} target
+   * @param {number} viewIndex
    * @return {boolean}
    */
-  getViewport(sessionId, eye, layer, target) { throw new Error('Not implemented'); }
+  getViewport(sessionId, eye, layer, target, viewIndex) { throw new Error('Not implemented'); }
 
   /**
    * @param {XREye} eye
+   * @param {number} viewIndex
    * @return {Float32Array}
    */
-  getProjectionMatrix(eye) { throw new Error('Not implemented'); }
+  getProjectionMatrix(eye, viewIndex) { throw new Error('Not implemented'); }
```

Comment:
I think this is the right thing to do as an incremental change, but wanted to note that if we're plumbing in a `viewIndex` the `eye` parameter here and in `getViewport` _should_ eventually become moot. Computing these values based only on the `viewIndex` is definitely the Right Thing going forward.

Reply:
It would be a breaking API change for things that implement XRDevices, e.g. https://github.com/MozillaReality/WebXR-emulator-extension. Is that a problem? If not I'm happy to clean this up.

Reply:
XRDevice isn't part of the public WebXR API, so it should only be used internally.

Despite that, I think it's perfectly fine to leave as a follow up.

Reply:
I'll probably follow up with this. Which do you think would be better:
- Remove `eye`, pass only `viewIndex`
- Remove `eye`+`viewIndex` and pass `XRSpace` (or `XRViewSpace`?)